### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by AleksandrPok**

### CODE REVIEW

File: src/main/java/cinema/model/dto/OrderResponseDto.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package cinema.model.dto;
+
+import java.util.List;
+
+public class OrderResponseDto {
+    private Long id;
+    private List<Long> tickets;
```
Comment (Max-Pochepets) [inline src/main/java/cinema/model/dto/OrderResponseDto.java]:
```suggestion
    private List<Long> ticketIds;
```

File: src/main/java/cinema/model/dto/ShoppingCartResponseDto.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package cinema.model.dto;
+
+import java.util.List;
+
+public class ShoppingCartResponseDto {
+    private Long userId;
+    private List<Long> tickets;
```
Comment (Max-Pochepets) [inline src/main/java/cinema/model/dto/ShoppingCartResponseDto.java]: same

**Review by Max-Pochepets — APPROVED** [review]

File: src/main/java/cinema/controller/AuthenticationController.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+
+    @Autowired
+    public AuthenticationController(AuthenticationService authenticationService) {
+        this.authenticationService = authenticationService;
+    }
+
+    @PostMapping("/register")
+    public void registration(@RequestBody UserRequestDto userRequestDto) {
```
Comment (KseniiaMakarova) [inline src/main/java/cinema/controller/AuthenticationController.java]:
```suggestion
    public void register(@RequestBody UserRequestDto userRequestDto) {
```
method names should be verbs

File: src/main/java/cinema/controller/ShoppingCartController.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+                                  UserService userService,
+                                  ShoppingCartMapper shoppingCartMapper) {
+        this.shoppingCartService = shoppingCartService;
+        this.movieSessionService = movieSessionService;
+        this.userService = userService;
+        this.shoppingCartMapper = shoppingCartMapper;
+    }
+
+    @PostMapping("/movie-session")
+    public void add(@RequestParam Long userId, @RequestParam Long movieSessionId) {
```
Comment (KseniiaMakarova) [inline src/main/java/cinema/controller/ShoppingCartController.java]:
```suggestion
    public void addSession(@RequestParam Long userId, @RequestParam Long movieSessionId) {
```

File: src/main/java/cinema/service/mappers/UserMapper.java [hunk 5]
```diff
@@ -0,0 +1,23 @@
+package cinema.service.mappers;
+
+import cinema.model.User;
+import cinema.model.dto.UserRequestDto;
+import cinema.model.dto.UserResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserMapper {
+    public User mapToEntity(UserRequestDto userRequestDto) {
```
Comment (KseniiaMakarova) [inline src/main/java/cinema/service/mappers/UserMapper.java]: do we need this method?

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**